bower.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "ember-couchdb-kit",
   "description": "An Ember.js adapter for Apache CouchDB",
+  "version": "1.0.2",
   "repository": {
     "type": "git",
-    "url": "git://github.com/zatvobor/ember-couchdb-kit.git"
+    "url": "git://github.com/ValidUSA/ember-couchdb-kit.git"
```

Comment:
This was my first github fork, and I didn't know what to do with this and version.  
We definitely prefer the changes to go upstream.  Our direction is to use _rev and _attachments instead of adding fields to the document.